backend/src/apis/IotaConnector.ts

```diff
@@ -0,0 +1,71 @@
+import { ClientBuilder } from '@iota/client'
+import { MessageWrapper } from '@iota/client/lib/types'
+
+import { CONFIG } from '@/config'
+import { backendLogger as logger } from '@/server/logger'
+
+const IotaClient = CONFIG?.IOTA_API_URL
+  ? new ClientBuilder().node(CONFIG.IOTA_API_URL).build()
+  : null
+
+/**
+ * send data message onto iota tangle
+ * use CONFIG.IOTA_COMMUNITY_ALIAS for index
+ * @param {string} message - the message as utf based string, will be converted to hex automatically from @iota/client
+ * @return {MessageWrapper|null} the iota message as json object with transaction data from iota or null if iota is disabled in config
+ */
+export const sendDataMessage = (message: string): Promise<MessageWrapper> | null => {
```

Comment:
While its ok to have this functionality like this here now, I recommend to define a specific API for Iota in the future in this file:

```
export const sendTransaction = (transaction: dbtransaction): Promise<boolean> => {
  if(CONFIG.IOTA)...
     iotaclient.sendMessage(toString(transaction))
}
```

For now this is acceptable, except all the comments which are not needed - I don't think we need a special capsul for the IOTA client for our `CONFIG.IOTA` switch - instead we write such function as `sendTransaction`,  which in turn then use the said switch